client/packages/common/src/intl/locales/en/common.json

```diff
@@ -1983,5 +1983,10 @@
   "warning.check-before-vaccinating": "Vaccination records from other stores may not be up to date. Ensure you have synced recently, and always confirm against the patient's physical vaccination card first.",
   "warning.field-not-parsed": "{{field}} not parsed",
   "warning.nothing-to-supply": "Nothing left to supply!",
-  "tooltip.defined-by-gs1-matrix": "Defined by GS1 datamatrix"
+  "tooltip.defined-by-gs1-matrix": "Defined by GS1 datamatrix",
+  "label.cannot-edit-invoice": "Cannot edit invoice",
+  "label.record-not-found": "Record not found",
+  "label.master-list-not-found": "Master list not found",
```

Comment:
these two already have translations, i think messages.record-not-found - probably best to reuse 👀

Reply:
Ah great thanks Lachè - I missed those